nucypher/utilities/prometheus/collector.py

```diff
@@ -15,14 +15,15 @@
 from eth_typing.evm import ChecksumAddress
 
 import nucypher
-from nucypher.blockchain.eth.actors import NucypherTokenActor
+from nucypher.blockchain.eth import actors
```

Comment:
Did the previous import for `NuCypherTokenActor` cause an issue?

Reply:
This is from 3159 which has already been merged